pkg/controller/framework/controller_test.go

```diff
@@ -335,7 +335,12 @@ func TestUpdate(t *testing.T) {
 		},
 	)
 
+	const threads = 3
+	testDoneWG.Add(threads * len(tests))
```

Comment:
You're right. Whoops.

This used to compile, but when @lavalamp suggested to reorganize the code I didn't see this error. The only sane way to fix this is to move the code back where it was, effectively undoing the suggestion from Daniel.